packages/web3-errors/src/errors/transaction_errors.ts

```diff
@@ -327,7 +327,7 @@ export class MissingGasError extends InvalidValueError {
 			}, maxPriorityFeePerGas: ${value.maxPriorityFeePerGas ?? 'undefined'}, maxFeePerGas: ${
 				value.maxFeePerGas ?? 'undefined'
 			}`,
-			'"gas" is missing',
+			'Missing properties in transaction, either define "gas" and "gasPrice" for type 0 transactions or "gas", "maxPriorityFeePerGas" and "maxFeePerGas" for type 2 transactions',
```

Comment:
Not suggesting we revert this change, but this is technically a breaking change since `'"gas" is missing'` was the error message thrown in `1.x` (and current `4.x`)